tag_database

```diff
@@ -17,6 +17,7 @@ bindKey:function,object,intermediate
 binomialCoefficient:math,intermediate
 bottomVisible:browser,intermediate
 btoa:node,string,utility,beginner
+bubbleSort:uncategorized
```

Comment:
As the snippet will be moved to archive, tags are probably not needed, so the scripts will automatically handle these. Nothing to worry about in this file.

Reply:
Ok.